.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: node_js
+node_js:
+  - "node"
+  - "6"
+  - "8"
+install:
+  - for d in runtime-version ssr ; do cd $d ; npm install || exit ; cd - ; done
```

Comment:
Can also generate list of directories that contain `package.json` via `find . -depth 2 -name 'package.json' -exec dirname \{\} \;` but it's maybe a bit magical/weird when there's only two to iterate through. (Another option is just to copy and paste the npm install/test lines, which will also avoid the need for the strange `|| exit`.)

Reply:
How about adding a script in the root project that does this?

Reply:
That's possible, how about `.travis/install` and `.travis/script` (to match the keys)? I don't think there's a convention, unfortunately.

Reply:
Now moved to `.travis/{install,script}`. `find` doesn't work because not cross platform enough but the shell script equivalent is okay.